monoio-http/src/common/body.rs

```diff
@@ -94,6 +104,81 @@ impl<T: Body<Data = Bytes>> BodyExt for T {
             }
         }
     }
+
+    fn decode_content(self, encoding: String) -> Self::BytesFuture {
```

Comment:
Encoding type as a String is a little bit strange... Make it as a enum is better.

Reply:
headers.get() returns `HeaderValue`, easier to  handle it as a string or &str. Using &str creates some lifetimes issue so I used String for now